airbyte-cdk/python/airbyte_cdk/logger.py

```diff
@@ -60,7 +73,7 @@ class AirbyteLogFormatter(logging.Formatter):
 
     def format(self, record: logging.LogRecord) -> str:
         """Return a JSON representation of the log message"""
-        airbyte_level = self.level_mapping.get(record.levelno, "INFO")
+        airbyte_level = self.level_mapping.get(record.levelno, Level.INFO)
```

Comment:
I didn't have to fix this to make the change work but while I was at it, this will avoid serpyco serialization issues